torch/csrc/jit/backends/backend_detail.h

```diff
@@ -58,11 +58,15 @@ TORCH_API void registerBackendPreprocessFunction(
     const std::string& name,
     const BackendPreprocessFunction& preprocess);
 
-TORCH_API bool hasBackendPreprocessFunction(const std::string& name);
+bool hasBackendPreprocessFunction(const std::string& name);
 
-TORCH_API BackendPreprocessFunction
-getBackendPreprocessFunction(const std::string& name);
+BackendPreprocessFunction getBackendPreprocessFunction(const std::string& name);
```

Comment:
Why is `TORCH_API` no longer needed? Is it because these functions are used only in `libtorch` after moving the codegen to `backend_detail.cpp`, and no longer used in `libtorch_python`?

Reply:
Yep, I originally added the TORCH_API because of the split but now shouldn't be needed.

Martin can confirm that's the case now.

Reply:
It's referenced by codegen_backend_module, which is already marked as ```TORCH_API```